projects/storefrontlib/src/cms-components/myaccount/order/order-details/order-detail-items/order-detail-items.component.html

```diff
@@ -1,6 +1,6 @@
 <ng-container *ngIf="order$ | async as order">
   <ng-container
-    *ngIf="order.consignments?.length || order.unconsignedEntries?.length"
+    *ngIf="order.consignments?.length || order.unconsignedEntries?.length || order.replenishmentOrderCode"
```

Comment:
The fix works well! Would it be possible to add a unit test to lock down the new logic?